### PR TITLE
add responsive landing page #63 

### PR DESCRIPTION
**Pull request by KRRISHSINGH08**
@0silverback0, Please merge this PR under the issue #63 and label it with hacktoberfest. Thank you

### CODE REVIEW

File: Responsive Ecommerce Landing Page/index.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+<body>
+    <div id="wrapper">
+
+        <!-- Navbar -->
+        <nav>
+            <div class="nav-left">
+                <a href="">KS.</a>
+            </div>
+            <div class="nav-mid">
+                <a href="#Home">Home</a>
```
Comment (0silverback0) [inline Responsive Ecommerce Landing Page/index.html]: this should link to the repo landing page /index.html then add your information in the repos index.html so your project and github can be view from the repo landing page

**Review by 0silverback0 — CHANGES_REQUESTED** [review]

**Comment by KRRISHSINGH08** [comment]
Please have a look @0silverback0

File: Responsive Ecommerce Landing Page/index.html (hunk 1 above)
Comment (0silverback0) [reply Responsive Ecommerce Landing Page/index.html]: line 21 should be href="/index.html" for ease of navigation

Comment (0silverback0) [reply Responsive Ecommerce Landing Page/index.html]: also in the root directory add your information so that your site is visible on the home page

**Comment by KRRISHSINGH08** [comment]
@0silverback0 I have added link to index.html in line 21 and also my info in the footer section. I think this will do

**Comment by 0silverback0** [comment]
merged!

**Pull request merged**